plugin-openshift/che-plugin-openshift-ext-server/src/main/java/org/eclipse/che/ide/ext/openshift/server/inject/OpenshiftModule.java

```diff
@@ -40,6 +41,7 @@ protected void configure() {
         bind(TemplateService.class);
         bind(ProjectService.class);
         bind(BuildService.class);
+        bind(OpenshiftApiInfoService.class);
```

Comment:
I think that we are using full *FQN* in cases like this, aren't we @skabashnyuk ?